pkgs/applications/editors/vim/plugins/generated.nix

```diff
@@ -9640,6 +9640,18 @@ final: prev:
     meta.homepage = "https://github.com/filipdutescu/renamer.nvim/";
   };
 
+  render-markdown = buildVimPlugin {
+    pname = "render-markdown";
```

Comment:
Shouldn't this be `markdown.nvim` however ?

Reply:
I thought it should match the alias. That's what samodostal-image-nvim does.

Reply:
Ok my bad then.